static/js/components/NewShift.jsx

```diff
@@ -82,6 +107,11 @@ const NewShift = () => {
         type: "string",
         title: "Shift's description",
       },
+      multipleDays: {
+        type: "boolean",
+        title: "Shift repeats everyday",
+        default: false,
+      },
```

Comment:
This was a little confusing for me to figure out what everyday means exactly. 
You should add in the description (or tooltip) that the shift will repeat during the days specified and that in each day separately the shift starts and ends based on the hh:mm:ss given.